xmpp.go

```diff
@@ -600,6 +601,17 @@ func (c *Client) Recv() (stanza interface{}, err error) {
 		case *clientPresence:
 			return Presence{v.From, v.To, v.Type, v.Show, v.Status}, nil
 		case *clientIQ:
+			//fmt.Printf("$+v", val)
```

Comment:
Thanks. Remove this line.

BTW, I'm not well known about this feature. There's not users who want to implement replying ping them-selfs?